jpo-asn-runtime/src/main/java/us/dot/its/jpo/asn/runtime/types/IA5String.java

```diff
@@ -0,0 +1,21 @@
+package us.dot.its.jpo.asn.runtime.types;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+
+/**
+ * Character range = 0..127, UPER encoded with 7 bits per character
+ * Ref: ITU-T X.691 (02/2021) Section 30
+ */
+public class IA5String extends Asn1CharacterString {
+
+    public IA5String(int minLength, int maxLength) {
+        super(minLength, maxLength, 7);
+    }
+
+    @JsonCreator
+    public IA5String(String value) {
+        this(0, Integer.MAX_VALUE);

```

Comment:
question: similar here, should we really be setting it to int.max?

Reply:
This constructor is for the case where we use a raw IA5String with no length constraint.  It's not the most common case, but there are a few in the spec.

Reply:
ok, and we're good with them being potentially this large?

Reply:
> ok, and we're good with them being potentially this large?

Yes and no.  It's possible to define a character string with unbounded length in ASN.1 so we want to be able to represent that.  But it's not considered a best practice to define an ASN.1 spec that way.

Reply:
> > ok, and we're good with them being potentially this large?
> 
> Yes and no. It's possible to define a character string with unbounded length in ASN.1 so we want to be able to represent that. But it's not considered a best practice to define an ASN.1 spec that way.

Same comment applies with octet strings btw.